src/components/WidgetKeyboards/WidgetKeyboardsPresentation.js

```diff
@@ -40,9 +40,9 @@ function WidgetKeyboardsPresentation({ header, urls }) {
               />
             </a>
           </div>
-          <div className="col-span-1 flex items-center">
+          <div className="flex items-center">
             <img
-              className="h-32 lg:h-52 w-auto shadow-2xl rounded-lg mx-auto border-8 border-white float-right "
```

Comment:
Can we scale or adjust the layout in mobile rather than hiding the logo altogether?